.taskcluster.yml

```diff
@@ -0,0 +1,315 @@
+---
```

Comment:
This is just a copy of the ones in Fenix and Firefox-iOS, with the proper renaming:
https://github.com/mozilla-mobile/firefox-ios/blob/main/.taskcluster.yml